internal/pkg/config/env.go

```diff
@@ -40,7 +40,10 @@ type CustomizeEnv struct {
 }
 
 // IsEmpty returns if CustomizeEnv is an empty struct.
-func (c CustomizeEnv) IsEmpty() bool {
+func (c *CustomizeEnv) IsEmpty() bool {
```

Comment:
Ooh, is this a chance to use `IsZero`?

Reply:
that's a good idea! Maybe we can do it as a follow-up PR? I didn't add any new `IsEmpty` methods so I didn't refactor for `IsZero`